Song/SongWriter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song
+{
+    public class SongWriter
+    {
+        public string Sing(List<string> animals, List<string> animalLines)
+        {
+            VerseWriter verserWriter = new VerseWriter();
+
+            if (animals.Count == 1) 
+            {
+                return verserWriter.WriteVerse(animals, animalLines);
+            }
```

Comment:
Not needed. If you move adding new line within the start of for loop iteration, it perfectly covers this.